src/pages/404.js

```diff
@@ -1,13 +1,8 @@
 import React from 'react'
-import Layout from '../components/Layout'
+import NotFound from '../components/NotFound'
 
-const NotFoundPage = () => (
-  <Layout>
-    <div>
-      <h1>NOT FOUND</h1>
-      <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
-    </div>
-  </Layout>
-)
+const NotFoundPage = () => {
+  return <NotFound />
```

Comment:
Why not to add the `NotFound` to here instead? Feels like this will be the only place where it will exists.

Reply:
I did it that way in order to have the styles in the same folder, but, I can move it to the global styles on its own file